zulip_bots/zulip_bots/zulip_bot_output.py

```diff
@@ -75,7 +77,7 @@ def main():
         sys.exit(1)
 
     with patch('zulip.Client') as mock_client:
-        mock_bot_handler = ExternalBotHandler(mock_client, bot_dir)
+        mock_bot_handler = ExternalBotHandler(mock_client, bot_dir)  # type: Any
```

Comment:
Is this necessary since this object gets 'monkey patched' later?

Reply:
I think so, otherwise I get several `Cannot assign to a method` errors right below (and many others later).